src/index.js

```diff
@@ -260,6 +260,13 @@ class Manager {
         
         // need to rename binary?
         if (downloadCfg.bin) {
+          try {
+            var linkPath = path.join(unpackFolder, activeCli.bin);
```

Comment:
Let's refactor this into a `const` variable above and then we can re-use below too for when we create the symlink.